core/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM rust:1.64.0
+ 
+WORKDIR /usr/src/myapp
```

Comment:
/usr/src/app for coherence with the other file?

Reply:
Or are they run on the same docker in which case it'd nice to use front and dust-api I guess?